app/views/disbursements/_select_organization.html.erb

```diff
@@ -0,0 +1,42 @@
+<%# locals: (form:, field_name:, disabled:, events:, default_event:, hide_balances: false, sending: false, receiving: false) %>
+
+<div class="field" data-controller="organization-select">
+    <%= form.label field_name, sending ? "From" : "To" %>
+    <select name="disbursement[<%= field_name %>]" id="disbursement_<%= field_name %>" <%= "disabled" if disabled %> class="bg-[transparent] dark:bg-darkless input input--select flex items-center select-none cursor-pointer disabled:cursor-default" data-organization-select-target="dropdown">
```

Comment:
Is our intention to be able to use this for things that aren't disbursements?

Reply:
For now, let's leave it as is. It would be cool to expand later but I don't want to DRY it too much preemptively.